src/subgraph/fully-connected.c

```diff
@@ -994,6 +1003,7 @@ enum xnn_status xnn_define_fully_connected(
     }
 
     switch (bias_value->datatype) {
+      case xnn_datatype_fp16:
```

Comment:
Not allow this?

Reply:
It will get caught by the follow up logic. I was planning to release a follow up pr to allow passing fp16 weights, so I think it's fine to leave here.